netman/adapters/switches/juniper/base.py

```diff
@@ -294,17 +295,17 @@ def set_interface_native_vlan(self, interface_id, vlan):
         if actual_port_mode is ACCESS:
             raise InterfaceInWrongPortMode("access")
         elif actual_port_mode is None:
-            update_attributes.append(self.custom_strategies.set_interface_port_mode_update_element("trunk"))
+            trunk_mode = self.custom_strategies.set_interface_port_mode_update_element("trunk")
```

Comment:
Reading this line i expect trunk_mode to be True or False... seems missleading

the method set_interface_port_mode_update_element seems to be returning an update element maybe the variable name could reflect that